Sources/Web3Inbox/WebView/WebViewFactory.swift

```diff
@@ -4,24 +4,34 @@ import WebKit
 final class WebViewFactory {
 
     private let host: String
-    private let webviewSubscriber: WebViewRequestSubscriber
+    private let chatWebviewSubscriber: WebViewRequestSubscriber
+    private let pushWebviewSubscriber: WebViewRequestSubscriber
 
-    init(host: String, webviewSubscriber: WebViewRequestSubscriber) {
+    init(
+        host: String,
+        chatWebviewSubscriber: WebViewRequestSubscriber,
+        pushWebviewSubscriber: WebViewRequestSubscriber
+    ) {
         self.host = host
-        self.webviewSubscriber = webviewSubscriber
+        self.chatWebviewSubscriber = chatWebviewSubscriber
+        self.pushWebviewSubscriber = pushWebviewSubscriber
     }
 
     func create() -> WKWebView {
         let configuration = WKWebViewConfiguration()
         configuration.allowsInlineMediaPlayback = true
         configuration.userContentController.add(
-            webviewSubscriber,
-            name: WebViewRequestSubscriber.name
+            chatWebviewSubscriber,
+            name: WebViewRequestSubscriber.chat
+        )
+        configuration.userContentController.add(
+            pushWebviewSubscriber,
+            name: WebViewRequestSubscriber.push
         )
         let webview = WKWebView(frame: .zero, configuration: configuration)
         let request = URLRequest(url: URL(string: host)!)
         webview.load(request)
-        webview.uiDelegate = webviewSubscriber
+        webview.uiDelegate = chatWebviewSubscriber
```

Comment:
@flypaper0 here was your most recent change in w3i and I am not sure what it was fixing. Please confirm that it's no breaking anything.